recipes/huracanpy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "huracanpy" %}
+{% set version = "1.2.0" %}
+{% set python_min = "3.9" %}
```

Comment:
You can remove this since it already matches the global conda-forge pinning of `python_min`